pkg/kubelet/dockershim/helpers.go

```diff
@@ -380,11 +381,33 @@ func ensureSandboxImageExists(client libdocker.Interface, image string) error {
 		return nil
 	}
 	if !libdocker.IsImageNotFoundError(err) {
-		return fmt.Errorf("failed to inspect sandbox image %q: %v", image, err)
+		return fmt.Errorf("failed to inspect pod infra container image %q: %v", image, err)
```

Comment:
Don't change this. We've retired the name "pod infra container image", and opt to use the "sandbox image" for CRI.